### DISCUSSION TITLE
Is there a specific event triggered when power is turned on through Hue emulation?

### DISCUSSION THREAD

**Discussion by Stevehans** [arendst/Tasmota#12761] [Ideas]
I haven't searched the source, but are there specific triggers for commands sent through the Hue emulation?

**Comment by ascillato2** [comment]
Sorry, No, there isn't. Changing this from Q&A to Feature Request.

**Reply by Stevehans** [reply]
My specific use case would be for my light with PIR, I want it on between sunset and 11pm, PIR activated between 11pm and sunrise.

However, if it were turned on by Alexa I'd want to override the PIR, hence ideally wanting specific Hue events.  I guess I could create a dummy relay and use that to drive the Alexa integration, but this would be easier.
